widgets/common/src/DynamicDropdown.jsx

```diff
@@ -33,12 +35,17 @@ function DynamicDropdown({
     const [hasMore, setHasMore] = useState(true);
     const [currentPage, setCurrentPage] = useState(-1);
     const [searchQuery, setSearchQuery] = useState('');
-    const [loaderVisible, setLoaderVisible] = useState(false);
+    const [shouldLoadMore, setShouldLoadMore] = useState(false);
```

Comment:
```suggestion
    const [shouldLoadMore, setShouldLoadMore] = useState(!!prefetch);
```

would probably allow us to remove https://github.com/cloudify-cosmo/cloudify-stage/blob/6d2ed65d065b058601fcd6c1f680a9a46f99b0dc/widgets/common/src/DynamicDropdown.jsx#L84-L86 and slightly reduce the complexity of this component :slightly_smiling_face:

Reply:
Good idea. Changed. 

WIth a small change, as I think in case of boolean value we don't need `!!`. Or am I missing something?

Reply:
Yup, you're right :slightly_smiling_face: I thought `prefetch` can be undefined, but it has a default value

Looks good :+1: